docs/100-upgrade/migration-guides/3.4-3.5.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 3.4 -> 3.5
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.4
+  to 3.5
+---
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/\*@3.5.0"
+```
+
+## Configuration updates
+
+In this new release, we've introduced some changes on how some configurations
+are set.
+
+### Cache
+
+We've introduce new cache related to the `config` object in `FrontCommerceApp`.
+These are now defined with a config provider, it allow you to change the
+behaviour of the configuration computing.
+
+| Config path                     | Env Variable                                |
+| ------------------------------- | ------------------------------------------- |
+| `config.cache.apiToken`         | `FRONT_COMMERCE_CACHE_API_TOKEN`            |
+| `config.cache.ignoreCacheRegex` | `FRONT_COMMERCE_BACKEND_IGNORE_CACHE_REGEX` |
+
+### Magento 1/2
+
+We have removed the `makeConfigFromEnv` helper in favor of values available in
```

Comment:
**nitpick:** this helper was a private internal code. It wasn't public API.

**suggestion:** don't mention it, and rephrase the paragraph into something like (feel free to adapt :sweat_smile:):
> We also made it possible to access Magento OAuth configurations via